wasmplugin/rules/coraza.conf-recommended.conf

```diff
@@ -41,11 +41,13 @@ SecRule REQUEST_HEADERS:Content-Type "^application/json" \
 # to the size of data, with files excluded. You want to keep that value as
 # low as practical.
 #
+# Running as a Wasm plugin, we expect Limit equal to MemoryLimit: it would be prevented buffering request body to files anyways.
+
 SecRequestBodyLimit 13107200
 
-SecRequestBodyInMemoryLimit 131072
+SecRequestBodyInMemoryLimit 13107200
```

Comment:
Is this change connected to the 1gb?

Reply:
No, it is about:
- Sets `RequestBodyLimit` equal to `BodyInMemoryLimit`.  As stated in Coraza itself: [TinyGo MemoryLimit should be equal to Limit](https://github.com/corazawaf/coraza/blob/2624bf31de378fffc793bdc738240062f7fdf6f1/internal/corazawaf/body_buffer.go#L74)

Not being able to split the request between memory and then into a file after a certain limit, for coraza-proxy-wasm the two limits should be the same